### ISSUE TITLE
"geneos init" command not returning a descriptive error message when using an invalid password

### ISSUE THREAD

**Issue by gadvincula** [ITRS-Group/cordial#77]
I tried running the "geneos init demo -u <account@email.com>" command with an incorrect password but the error message returned was not suggestive of that error otherwise. Please see screenshot below:

user@server_hostname:~$ geneos init demo -u **********@*********.com
Password: <incorrect password was used here>
gateway "Demo Gateway" added, port 7039
netprobe "localhost" added, port 7036
2023-05-02T05:01:45Z FATAL: tools/geneos/cmd/add.go:154 cmd.addInstance() > error="chdir /home/sysad1/geneos/packages/webserver/active_prod/config: no such file or directory"

**Comment by pgalbavy-itrs** [comment]
Yes, I see this. The program should exit much earlier in the init cycle and ideally try to revert changes if possible.

Will investigate and fix.

**Comment by pgalbavy-itrs** [comment]
Poor coding - none of the install routines (in fact none of the hand-off init routines) has their error returns checked. This how now been fixed:

```bash
$ geneos init demo -u pgalbavy@itrsgroup.com
Password: 
Error: cannot access gateway package at "https://resources.itrsgroup.com/download/latest/Gateway+2?os=linux" version latest: 401 Unauthorized
```

fix in v1.5.0-dev, closing this issue for now.
